### DISCUSSION TITLE
next app is being served from only build

### DISCUSSION THREAD

**Discussion by rabius-sunny** [vercel/next.js#29993] [Help] [answered]
Hi there, I've a next app. When I do a `yarn start/next start` there, it throws me an error with `Could not find a production build`. But after building the app, it only serves that built project, I am not getting any live interaction on browser whenever I make a change to my code.

## Updated:
I think this was my mistake. I've to run `yarn dev` instead of `yarn start`.

**Comment by jamesmosier** [comment, marked as answer]
`yarn dev` should be used for development while `yarn start` is meant to be run in production. Your code will not live reload when running `yarn start`.
